corehq/apps/data_interfaces/views.py

```diff
@@ -470,6 +475,11 @@ def post(self, request, *args, **kwargs):
             return self.get(request, *args, **kwargs)
         return self.paginate_crud_response
 
+    @use_bootstrap3
+    @use_knockout_js
```

Comment:
please pull / rebase latest master into this branch and remove `use_knockout_js` :)